chasm/lib/scheduler/scheduler.go

```diff
@@ -64,15 +66,51 @@ func NewScheduler(
 		cacheConflictToken: scheduler.InitialConflictToken,
 		Backfillers:        make(chasm.Map[string, *Backfiller]),
 	}
+	sched.Info.CreateTime = timestamppb.New(ctx.Now(sched))
 
 	invoker := NewInvoker(ctx, sched)
 	generator := NewGenerator(ctx, sched, invoker)
 	sched.Invoker = chasm.NewComponentField(ctx, invoker)
 	sched.Generator = chasm.NewComponentField(ctx, generator)
 
+	// Create backfillers to fulfill initialPatch.
+	sched.handlePatch(ctx, patch)
+
 	return sched
 }
 
+// handlePatch creates backfillers to fulfill the given patch request.
+func (s *Scheduler) handlePatch(ctx chasm.MutableContext, patch *schedulepb.SchedulePatch) {
+	if patch != nil {
+		if patch.TriggerImmediately != nil {
+			s.NewImmediateBackfiller(ctx, patch.TriggerImmediately)
+		}
+
+		for _, backfill := range patch.BackfillRequest {
+			s.NewRangeBackfiller(ctx, backfill)
+		}
+	}
+}
+
+// Create initializes a new Scheduler for CreateSchedule requests.
+func Create(
+	ctx chasm.MutableContext,
+	req *schedulerpb.CreateScheduleRequest,
+) (*Scheduler, *schedulerpb.CreateScheduleResponse, error) {
+	// TODO: namespace name should be resolved from namespace_id via namespace registry
+	sched := NewScheduler(
+		ctx,
+		"", // TODO: should be namespace name
+		req.NamespaceId,
```

Comment:
Can't the scheduler get this information from CHASM? The execution / component ref should include all of this information.